src/Views/Shared/Time/TimePeriodNoAppointments.cshtml

```diff
@@ -1,3 +1,12 @@
-@model string
+@model form_builder.Models.Time.TimePeriod
 
-<p class="smbc-body">@Model</p>
\ No newline at end of file
+@{
+    var timeOfDay = @Model.TimeOfDay.ToString().ToLower();
+}
+<div class="govuk-warning-text">
+    <span class="govuk-warning-text__icon govuk-!-margin-top-6" aria-hidden="true">!</span>
+    <strong class="govuk-warning-text__text govuk-!-margin-left-4">
+        <span class="govuk-warning-text__assistive">Warning</span>
+        There are no available @timeOfDay @Model.BookingType.ToLower() on this date, please select a @timeOfDay time slot or a different date.
```

Comment:
If we are reusing `@timeOfDay` for this message, the warning will read "There are no available morning appointments on this date, please select a morning time slot or a different date.". Looking at the mock up this is what UX have given us for the morning message, however, this makes no sense to a user as they are being told there are no morning slots and then to choose a morning slot.

This logic won't work for what UX have supplied for the afternoon message, which I feel is the intended behaviour: "There are no available afternoon appointments on this date, please select a morning time slot or a different date". This inverts the time of day so it makes sense to a user that the afternoon appointments are gone and they need to select a morning appointment.

Also there is no full stop at the end of the message in the mock up 👍

Reply:
Thanks for the feedback. I have added the revert logic for morning and afternoon with the full stop removed.